content/advanced/_index.en.md

```diff
@@ -338,11 +338,21 @@ Or a remote URI:
 
 ### Environment variables
 
-When LocalAI runs in a container, there are additional environment variables available that modify the behavior of LocalAI on startup:
+When LocalAI runs in a container,
+there are additional environment variables available that modify the behavior of LocalAI on startup:
 
 | Environment variable       | Default | Description                                                                                                |
 |----------------------------|---------|------------------------------------------------------------------------------------------------------------|
 | `REBUILD`                  | `true`  | Rebuild LocalAI on startup                                                                                 |
 | `BUILD_TYPE`               |         | Build type. Available: `cublas`, `openblas`, `clblas`                                                      |
 | `GO_TAGS`                  |         | Go tags. Available: `stablediffusion`                                                                      |
```

Comment:
Could we add ``tts`` here?

Reply:
Oops that did not work how I was thinking it would work, please dont mind that comment